### PR TITLE
[CARBONDATA-794] Numeric dimension column value should be validated for bad record

### PR DESCRIPTION
**Pull request by mohammadshahidkhan**
Added bad record logger for Numeric column added in DICTIONARY_INCLUDE

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1226/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1228/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1229/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1234/

File: processing/src/main/java/org/apache/carbondata/processing/newflow/converter/impl/DictionaryFieldConverterImpl.java [hunk 1]
```diff
@@ -94,8 +97,19 @@ public DictionaryFieldConverterImpl(DataField dataField,
   @Override public void convert(CarbonRow row, BadRecordLogHolder logHolder)
       throws CarbonDataLoadingException {
     try {
-      String parsedValue = DataTypeUtil.parseValue(row.getString(index), carbonDimension);
-      if (null == parsedValue || parsedValue.equals(nullFormat)) {
+      String dimensionValue = row.getString(index);
+      if (dimensionValue == null || dimensionValue.equals(nullFormat)) {
```
Comment (gvramana) [inline processing/src/main/java/org/apache/carbondata/processing/newflow/converter/impl/DictionaryFieldConverterImpl.java]: write comment that this is alter case

**Comment by gvramana** [comment]
LGTM

**Pull request closed without merging**